dask_cuda/tests/test_utils.py

```diff
@@ -11,3 +13,29 @@ def test_get_n_gpus():
         assert get_n_gpus() == 3
     finally:
         del os.environ["CUDA_VISIBLE_DEVICES"]
+
+
+@pytest.mark.parametrize(
+    "params",
+    [
+        {
+            "input": [1152920405096267775, 0],
+            "output": [i for i in range(20)] + [i + 40 for i in range(20)],
+        },
+        {
+            "input": [17293823668613283840, 65535],
+            "output": [i + 20 for i in range(20)] + [i + 60 for i in range(20)],
+        },
+        {"input": [18446744073709551615, 0], "output": [i for i in range(64)]},
+        {"input": [0, 18446744073709551615], "output": [i + 64 for i in range(64)]},
+    ],
+)
+def test_bitmask_to_list(params):
+    assert bitmask_to_list(params["input"]) == params["output"]
+
+
+def test_cpu_affinity():
+    for i in range(get_n_gpus()):
+        affinity = get_cpu_affinity_list(i)
+        os.sched_setaffinity(0, affinity)
+        assert list(os.sched_getaffinity(0)) == affinity
```

Comment:
This seems to test the `os.sched_setaffinity`/`os.sched_getaffinity` functionality more than it tests `get_cpu_affinity_list`?

Reply:
Well, both. I'm actually verifying that what I'm setting is indeed workable. If I pass dummy values the result will be wrong and we can't predict the affinity of the machine running the test, so it's probably the best to check what we expected to set got set.

Reply:
Ah, good point